nemo/nemo/utils/exp_logging.py

```diff
@@ -67,7 +69,8 @@ def __init__(
             ckpt_dir=None,
             tb_dir=None,
             files_to_copy=None,
-            add_time=True):
+            add_time=True,
```

Comment:
Is there *any reason* for NOT using time as part of experiment directory name? (asking as I cannot see any reason for that)

If not, then please get rid of this option, the code will be much cleaner/readable.

Reply:
Adding time was broken before this PR. It can probably be removed now.

Reply:
On second thought. I want to add this back. When I continue training an experiment, I want to use the same logdir without it creating a new one with an additional time ending.
We can just change the default to True

Reply:
Ok, I like that use case, I was showing some time ago such an option in PyTorch Lightning:
https://williamfalcon.github.io/pytorch-lightning/Trainer/Checkpointing/#restoring-training-session

Are we supporting "experiment continuation" on a "system level", i.e. are we saving "the trainer state"?

Reply:
>> "are we saving "the trainer state"?
- Yes, checkpointcallback can do that